config/302-filter.yaml

```diff
@@ -122,26 +122,25 @@ spec:
                 - required: [uri]
                 properties:
                   ref:
-                    description: Ref points to an Addressable.
+                    description: Reference to an addressable Kubernetes object to be used as the destination of events.
                     type: object
                     properties:
                       apiVersion:
-                        description: API version of the referent.
                         type: string
                       kind:
-                        description: 'Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-                        type: string
-                      name:
-                        description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
                         type: string
                       namespace:
-                        description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
-                          This is optional field, it gets defaulted to the object holding it if left out.'
                         type: string
```

Comment:
It looks like the 5 CRDs I mentioned had `description` attributes for `apiVersion`, `kind`, `namespace` and `name`.
I have a feeling that they are obvious and could be removed, but if we prefer to keep them I'll add them to the template and re-run the script. This will propagate descriptions to all source CRDs (and create a much larger PR).